apps/scan/frontend/src/components/printer_management/poll_worker_load_and_reprint_button.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react';
+import { Button, Modal, P, useQueryChangeListener } from '@votingworks/ui';
+import { assert, throwIllegalValue } from '@votingworks/basics';
+import { getPrinterStatus } from '../../api';
+
+function PollWorkerLoadPaperModal({
+  onClose,
+}: {
+  onClose: () => void;
+}): JSX.Element {
+  const printerStatusQuery = getPrinterStatus.useQuery();
+  const printerStatus = printerStatusQuery.data;
+  assert(printerStatus);
+  assert(printerStatus.scheme === 'hardware-v4');
+
+  const [hasRemovedPaperRollHolder, setHasRemovedPaperRollHolder] = useState(
+    printerStatus.state === 'cover-open'
+  );
+
+  // advance the user flow when we detect the printer was opened
+  useQueryChangeListener(printerStatusQuery, {
+    onChange: (newPrinterStatus) => {
+      assert(newPrinterStatus.scheme === 'hardware-v4');
+      if (newPrinterStatus.state === 'cover-open') {
+        setHasRemovedPaperRollHolder(true);
+      }
+    },
+  });
+
+  // user has not yet removed the paper roll holder
+  if (!hasRemovedPaperRollHolder) {
+    // we shouldn't need to handle errors in this initial state, because the
+    // modal shouldn't be open-able if the printer is in an error state
+    return (
+      <Modal
+        title="Remove Paper Roll Holder"
+        content={
+          <P>
+            Open the access door to reveal the printer. Press the green lever on
+            the paper roll holder to separate it from the printer.
+          </P>
+        }
+        actions={<Button onPress={onClose}>Cancel</Button>}
+      />
+    );
+  }
+
+  // user has removed the paper roll holder, we guide them through reloading the paper
+  switch (printerStatus.state) {
+    case 'error':
+      return (
+        <Modal
+          title="Printer Error"
+          content={<P>The printer has encountered an unexpected error.</P>}
+          actions={<Button onPress={onClose}>Close</Button>}
+        />
+      );
+    case 'cover-open':
+      return (
+        <Modal
+          title="Load New Paper Roll"
+          content={
+            <P>
+              Slide a new roll of paper onto the roll holder. Unroll enough
+              paper to pull it over the tear bar and toward you. Holding the end
+              of the paper with your thumbs, push the roll holder back onto the
+              printer so it clicks into place.
```

Comment:
For the instructions that are identical across the EM and PW experiences, would be nice to use shared constants so that they remain consistent